config.ru

```diff
@@ -1,15 +1,6 @@
 # This file is used by Rack-based servers to start the application.
 
-require ::File.expand_path('../config/environment', __FILE__)
-require 'rack/cors'
-
-NewRelic::Agent.manual_start
-
-use Rack::Cors do
-  allow do
-    origins '*'
-    resource '*', headers: :any, methods: [:get, :post, :put, :delete, :options]
-  end
-end
```

Comment:
we still need these lines (although the NR setup may change later per SRCH-2170)

Reply:
Restored.